src/decompiler.cc

```diff
@@ -16,40 +16,17 @@
 
 #include "src/decompiler.h"
 
-#include <algorithm>
-#include <array>
-#include <cassert>
-#include <cinttypes>
-#include <cstdarg>
-#include <cstdio>
-#include <iterator>
-#include <map>
-#include <numeric>
-#include <set>
-#include <string>
-#include <vector>
-#include <sstream>
-
-#include "src/cast.h"
-#include "src/common.h"
-#include "src/expr-visitor.h"
-#include "src/ir.h"
-#include "src/ir-util.h"
-#include "src/literal.h"
-#include "src/generate-names.h"
-#include "src/stream.h"
+#include "src/decompiler-ast.inl"
 
 #define WABT_TRACING 0
 #include "src/tracing.h"
 
 namespace wabt {
 
-namespace {
-
-struct Decompiler : ModuleContext {
+struct Decompiler {
```

Comment:
Why get rid of the anonymous namespace here?

Reply:
Since the code here and in `decompiler-ast.inl` need to be in the same namespace. I suppose wit would be possible to retain it here, but since I had to remove it in the other it seemed logical to not have it at all.